api/python/tests/integration/test_packages.py

```diff
@@ -1393,10 +1393,20 @@ def test_install(self):
                 'test/foo/foo',
             )))
 
+    def test_install_subpackage_deprecated_and_new(self):
+        pkg_name = 'Quilt/Foo'
+        bucket = 'my-test-bucket'
+        path = 'baz'
+        dest = 'package'
+
+        with pytest.warns(RemovedInQuilt4Warning):
+            with pytest.raises(ValueError):
```

Comment:
I'm not clear why we also raise a hard ValueError if thi sis a soft deprecation?

Reply:
We raise `ValueError` when the path is passed both via package name and the `path` parameter.

Reply:
i.e. it doesn't make sense to call `install('usr/pkg/path1', path='path2')`.